scripts/policy/Source_hsc.yaml

```diff
@@ -0,0 +1,278 @@
+funcs:
```

Comment:
Can you please put a docstring at the top of this file describing what it is for?